genocrowd/app.py

```diff
@@ -51,6 +57,7 @@ def create_app(config='config/genocrowd.ini', app_name='genocrowd', blueprints=N
     Flask
         Genocrowd Flask application
     """
+    print('yo')
```

Comment:
zis is debug, please remove t

Reply:
For logging, use `app.logger.debug` for debuging